zjit/src/hir/opt_tests.rs

```diff
@@ -7018,6 +7023,54 @@ mod hir_opt_tests {
         ");
     }
 
+    #[test]
+    fn counting_fancy_feature_use_for_fallback() {
```

Comment:
Let's do these in two separate tests please. Having sequential snapshots makes testing awkward/irritating